packages/lib/gpcircuits/turbo.json

```diff
@@ -8,12 +8,12 @@
         "../../../node_modules/circomkit/**",
         "../../../node_modules/circomlib/**",
         "../../../node_modules/@zk-kit/circuits/**",
-        "scripts/gen-test-artifacts.ts",
+        "scripts/gen-artifacts.ts",
         "circuits/**",
         "!circuits/test/**",
         "!circuits/main/**",
         "circomkit.json",
-        "circuits.json"
+        "testCircuits.json"
```

Comment:
circuits.json originally existed because it was needed by circomkit when running from the command-line.  It's not clear to me which set should be in the primary file for that purpose.  I'm also not clear whether unit tests depend on circuits.json or not.  Have you tested a case where the test and prod families aren't the same?  Can unit tests properly generate files which aren't in circuits.json?  If not, maybe it neesd to be a union of all circuits, while testCircuits.json and prodCircuits.json are the separate sets used for artifact generation.